src/test/java/com/hotels/beeju/core/HiveServer2CoreTest.java

```diff
@@ -18,18 +18,42 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
 
 import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.Arrays;
+import java.util.List;
 
 import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.hive.metastore.HiveMetaStoreClient;
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+import org.apache.hadoop.hive.metastore.api.NoSuchObjectException;
+import org.apache.hadoop.hive.metastore.api.Partition;
+import org.apache.hadoop.hive.metastore.api.SerDeInfo;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
 import org.apache.hive.service.Service;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 
 public class HiveServer2CoreTest {
 
-  private final BeejuCore core = new BeejuCore();
+  private static final String DATABASE = "my_test_db";
+  private final BeejuCore core = new BeejuCore(DATABASE);
   private final HiveServer2Core hiveServer2Core = new HiveServer2Core(core);
 
+  public TemporaryFolder temporaryFolder;
+
+  @Before
+  public void setup() {
+    temporaryFolder = new TemporaryFolder();
```

Comment:
Can this be moved to line 50 so we don't need this method?

Reply:
turns out we don't use the folder within the test soooo... 👋